src/Symfony/Bridge/Twig/NodeVisitor/Scope.php

```diff
@@ -42,7 +42,7 @@ public function __construct(Scope $parent = null)
     /**
      * Opens a new child scope.
      *
-     * @return Scope
+     * @return self
```

Comment:
This seems to be inconsistent with a lot of changes below.. where it goes to `static` for `new self`.